.Rbuildignore

```diff
@@ -36,3 +36,4 @@
 ^docs$
 ^pkgdown$
 ^\.github$
+^vignettes/articles$
```

Comment:
articles are like vignettes for pkgdown, but they're not included in the built R package.